### DISCUSSION TITLE
Unknown /app/xdg volume

### DISCUSSION THREAD

**Discussion by NGDM** [Taxel/PlexTraktSync#1570] [Support] [answered]
On Portainer I'm noticing a lot of unused /app/xdg volumes being created for this Docker.
It seems it's an undocumented volume for some type of caching? 

Now it just keeps creating new volumes (at every update I'm assuming) and it just piles up.
Should I just map this volume to a local folder, or is there some ENV I can set to make it map to the regular config folder?

Thanks for clarifying.

**Comment by glensc** [comment, marked as answer]
Added here:
- https://github.com/Taxel/PlexTraktSync/pull/472

Use named volume if you don't like Portainer behavior.

Also you can submit PR:

> 
> Contribute
> Looking for a way to contribute?
> 
> find issues with the [help-wanted](https://github.com/Taxel/PlexTraktSync/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22+sort%3Aupdated-desc) label
> improve documentation [docs-needed](https://github.com/Taxel/PlexTraktSync/issues?q=label%3A%22docs+needed%22+sort%3Aupdated-desc) label
> you can also just [create a pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request) to improve documentation
> ... more [developer contribution](https://github.com/Taxel/PlexTraktSync/blob/main/CONTRIBUTING.md) docs
